### DISCUSSION TITLE
kitefast variant

### DISCUSSION THREAD

**Discussion by jeremy-rutman** [OpenFAST/openfast#865] [Ideas]
Hi all
I'm interested in trying kitefast but would like to use something other than a makani-style rigid airship w. fuselage , rudder etc. - I'd rather try aerofoils and other kite shapes  - is this a major project , what would it entail?

Thanks 
Jeremy

**Comment by jjonkman** [comment]
Hi @jeremy-rutman,

The general configuration of an energy kite currently supported by KiteFAST is summarized in the online KiteFAST documentation here: https://kitefast.readthedocs.io/en/kitefast/source/user/kitembdynpreprocessor/index.html.  I gather based on your comment that the energy kite configuration you have in mind is different than what is outlined?  If so, I would guess that would be a significant effort to modify KiteFAST, but the level of effort likely depends on the configuration you have in mind.  Can you clarify?

Best regards,
